docs/kh2/file/type/areadata.md

```diff
@@ -41,7 +41,7 @@ When referring to _entrance_ it means that is the index where to spawn the chara
 | 18     | int   | Always 0 [*¹](#notes)
 | 1c     | byte  | [Area ID of a world](../../worlds.md)
 | 1d     | byte  | Entrance
-| 1e     | byte  | [World ID](../../worlds.md)
+| 1e     | byte  | Approach Direction Trigger
```

Comment:
Is there somewhere a description on what this is?

Reply:
It's basically determines if the event is triggered only when approaching from a specific direction or if it can be triggered by approaching it from any direction (even from above/below). An example of the former would be the room transition used in CoR skip where you land behind the trigger and have to move away from the exit for a bit in order to move to CoR: Depths. An example of the latter would be Cloud's mandatory cutscene in HB.

The above explanation can probably be worded more concisely, but I can't find the right words.

Reply:
If you can add that somewhere as a note then maybe other people can dive deeper into it or reword it.

Reply:
Explanation added.